src/main/java/mil/dds/anet/AnetApplication.java

```diff
@@ -386,18 +371,20 @@ public void run(AnetConfiguration configuration, Environment environment)
     final SubscriptionResource subscriptionResource = new SubscriptionResource(engine);
     final SubscriptionUpdateResource subscriptionUpdateResource =
         new SubscriptionUpdateResource(engine);
+    final AttachmentResource attachmentResource = new AttachmentResource(engine);
     final GraphQlResource graphQlResource = injector.getInstance(GraphQlResource.class);
     graphQlResource.initialise(engine, configuration,
         ImmutableList.of(reportResource, personResource, positionResource, locationResource,
             orgResource, taskResource, adminResource, savedSearchResource,
             authorizationGroupResource, noteResource, approvalStepResource, subscriptionResource,
-            subscriptionUpdateResource),
+            subscriptionUpdateResource, attachmentResource),
         metricRegistry);
 
     // Register all of the HTTP Resources
     environment.jersey().register(loggingResource);
     environment.jersey().register(adminResource);
     environment.jersey().register(homeResource);
+    environment.jersey().register(attachmentResource);
```

Comment:
If `attachmentResource` only handles GraphQL requests, it shouldn't be registered to jersey.